src/Plugin/CaptureUtility/ScreenshotCaptureUtility.php

```diff
@@ -83,66 +82,29 @@ public function missingDependencies() {
     return $missing_dependencies;
   }
 
-  // /**
-  //  * {@inheritdoc}
-  //  */
-  // public function addConfigFormFields(array $form, WebPageArchive $web_page_archive = NULL) {
-  //   // Default form options:
-  //   $config = [
-  //     "{$this->pluginId}" => FALSE,
-  //     "{$this->pluginId}_width" => 1280,
-  //   ];
-  //
-  //   // Look for set values.
-  //   if (isset($web_page_archive)) {
-  //     $instance = $web_page_archive->hasCaptureUtilityInstance($this->pluginId);
-  //     $config = $instance['config'];
-  //   }
-  //
-  //   // Setup form fields.
-  //   $form[$this->pluginId] = [
-  //     '#type' => 'checkbox',
-  //     '#title' => $this->t('Capture Screenshot?'),
-  //     '#description' => $this->t('If checked, this job will include download and compare screenshots.'),
-  //     // '#default_value' => $web_page_archive->isScreenshotCapturing(),
-  //     '#default_value' => $config[$this->pluginId],
-  //   ];
-  //   $form["{$this->pluginId}_width"] = [
-  //     '#type' => 'number',
-  //     '#title' => $this->t('Capture width (in pixels)'),
-  //     '#description' => $this->t('Specify the width you would like to capture.'),
-  //     // '#default_value' => $web_page_archive->isScreenshotCapturing(),
-  //     // '#default_value' => TRUE,
-  //     '#default_value' => $config["{$this->pluginId}_width"],
-  //   ];
-  //
-  //   return $form;
-  // }
-
   /**
    * {@inheritdoc}
    */
   public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
     $form['intro'] = [
       '#markup' => $this->t('Some instructions go here.'),
     ];
-    // $form['capture'] = [
-    //   '#type' => 'checkbox',
-    //   '#title' => $this->t('Capture Screenshot?'),
-    //   '#description' => $this->t('If checked, this archive will download and compare screenshots.'),
-    //   // '#default_value' => $web_page_archive->isScreenshotCapturing(),
-    //   // '#default_value' => $config[$this->pluginId],
-    //   // TODO: How to determine this?
-    //   '#default_value' => TRUE,
-    // ];
     $form['width'] = [
       '#type' => 'number',
       '#title' => $this->t('Capture width (in pixels)'),
       '#description' => $this->t('Specify the width you would like to capture.'),
-      // TODO: Get this value...
       '#default_value' => 1280,
```

Comment:
I think we should leave this todo in here for now.  If we edit a capture plugin, we're gonna want to retain the previously set value and this todo serves as a reminder to fix that.